src/services/network-modification-types.ts

```diff
@@ -37,3 +48,307 @@ export interface VscModificationInfo {
+    isReactiveCapabilityCurveOn?: boolean;
+    minQ?: number | null;
+    maxQ?: number | null;
+    reactiveCapabilityCurve?: ReactiveCapabilityCurvePointsData | undefined;
+    targetP: number;
+    targetQ: number;
+    participate: boolean;
+    droop: number;
+    isUpdate?: boolean;
+    properties?: Property[];
+}
+
+export interface LoadModificationInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    modificationUuid?: string;
+    id: string;
+    name: string | null;
+    loadType: string;
+    voltageLevelId?: string;
+    busOrBusbarSectionId?: string;
+    connectionName?: string | null;
+    connectionDirection?: string | null;
+    connectionPosition?: string | null;
+    terminalConnected?: boolean | null;
+    p0: number;
+    q0: number;
+    isUpdate?: boolean;
+    properties?: Property[];
+}
+
+export interface ShuntCompensatorModificationInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    shuntCompensatorId: string;
+    shuntCompensatorName: string | null;
+    maxSusceptance: number | null;
+    maxQAtNominalV: number | null;
+    shuntCompensatorType: string;
+    voltageLevelId?: string;
+    busOrBusbarSectionId?: string;
+    sectionCount: number;
+    maximumSectionCount: number;
+    connectivity?: any;
+    isUpdate?: boolean;
+    modificationUuid?: string;
+    connectionDirection?: string | null;
+    connectionName?: string | null;
+    connectionPosition?: string | null;
+    terminalConnected?: boolean | null;
+    properties?: Property[];
+}
+
+export interface GeneratorModificationInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    generatorId: string;
+    name: string | null;
+    energySource: string;
+    minP: number;
+    maxP: number;
+    ratedS?: number | null;
+    targetP: number | null;
+    targetQ: number | null;
+    voltageRegulation: boolean;
+    targetV: number | null;
+    qPercent: number | null;
+    voltageLevelId?: string;
+    busOrBusbarSectionId?: string;
+    isUpdate?: boolean;
+    modificationUuid?: string;
+    plannedActivePowerSetPoint: number;
+    marginalCost: number;
+    plannedOutageRate: number;
+    forcedOutageRate: number;
+    directTransX: number;
+    stepUpTransformerX: number;
+    voltageRegulationType?: any;
+    regulatingTerminalId: string | null;
+    regulatingTerminalType: string | null;
+    regulatingTerminalVlId: string | null;
+    isReactiveCapabilityCurveOn?: boolean;
+    participate: boolean;
+    droop: number | null;
+    maxQ?: number | null;
+    minQ?: number | null;
+    reactiveCapabilityCurve?: ReactiveCapabilityCurvePointsData[] | undefined;
+    connectionDirection?: string | null;
+    connectionName?: string | null;
+    connectionPosition?: string | null;
+    terminalConnected?: boolean | null;
+    properties?: Property[];
+}
+
+export interface StaticVarCompensatorCreationInfo {
+    studyUuid: string;
+    nodeUuid: UUID;
+    staticCompensatorId: string;
+    staticCompensatorName: string | null | undefined;
```

Comment:
to be more concistent with the remaning definitions in this class
```suggestion
    staticCompensatorName?: string | null;
```

Reply:
Done